src/components/Developer/Developer.tsx

```diff
@@ -49,48 +65,66 @@ const geomodelFileName = Platform.select( {
   ios: Config.IOS_GEOMODEL_FILE_NAME,
   android: Config.ANDROID_GEOMODEL_FILE_NAME
 } );
+const boldClassname = ( line: string, isDirectory = false ) => classnames(
+  {
+    "text-red font-bold": line.includes( "MB" ),
+    "text-blue": isDirectory
+  }
+);
 
-/* eslint-disable i18next/no-literal-string */
-const Developer = (): Node => {
-  const fileSizes = useAppSize( );
+interface DirectorySizesProps {
+  directoryName: string;
+  directoryEntrySizes: DirectoryEntrySize[]
+}
 
-  const boldClassname = ( line, isDirectory = false ) => classnames(
-    {
-      "text-red font-bold": line.includes( "MB" ),
-      "text-blue": isDirectory
-    }
+/* eslint-disable i18next/no-literal-string */
+const DirectoryFileSizes = ( { directoryName, directoryEntrySizes }: DirectorySizesProps ) => {
+  const totalDirectorySize = formatSizeUnits( getTotalDirectorySize( directoryEntrySizes ) );
+  return (
+    <View key={directoryName}>
+      <H2>
+        File Sizes:
+        {" "}
+        {directoryName}
+      </H2>
+      <P>
+        <CODE optionalClassName={boldClassname( totalDirectorySize, true )}>
+          {`Total Directory Size: ${totalDirectorySize}`}
+        </CODE>
+      </P>
+      {directoryEntrySizes.map( ( { name, size } ) => {
+        const line = formatAppSizeString( name, size );
+        return (
+          <P key={name}>
+            <CODE optionalClassName={boldClassname( line )}>
+              {line}
+            </CODE>
+          </P>
+        );
+      } )}
+    </View>
   );
+};
 
-  const displayFileSizes = useCallback( ( ) => Object.keys( fileSizes ).map( directory => {
```

Comment:
Could you make more granular commits here, so that the process of changes becomes easier to review. The refactor of this code block into standalone component, i.e. copying it over into the new component, could have been one commit. So that the changes in this one commit are only copy paste and indentation changes. This makes it easier for me to see which pieces of the code are actually changed and which ones just copied over.

Reply:
We chatted about this, but reiterating: I'm trying to be more mindful of this going forward : )